src/actions/actionCanvas.tsx

```diff
@@ -304,12 +304,13 @@ export const actionErase = register({
       commitToHistory: true,
     };
   },
+  keyTest: (event) => event.code === CODES.E,
```

Comment:
Let's use `event.key` here to align

Reply:
I can but then I will have to convert it to `lower/Upper case` since the `key` will be different for upper and lower case but code is always same

Reply:
we use `event.key` for all toolbar tools, and we don't lowercase them (i.e. a `Shift-R` is different from `R`), so I wouldn't do that here either